test/helpers/replicate.js

```diff
@@ -6,29 +6,50 @@ const { promisify: p } = require('util')
 const pull = require('pull-stream')
 const pullMany = require('pull-many')
 const deepEqual = require('fast-deep-equal')
+const { replicate } = require('scuttle-testbot')
 
 /**
  * Fully replicates person1's metafeed tree to person2 and vice versa
  */
 module.exports = async function replicate(person1, person2) {
   // Establish a network connection
-  const conn = await p(person1.connect)(person2.getAddress())
+  const conn = await p(person1.connect)(person2.getAddress()).catch(
+    console.error
+  )
+
+  const isSync = await ebtReplicate(person1, person2).catch((err) =>
+    console.error('Error with ebtReplicate:\n', err)
+  )
+
+  await p(conn.close)(true).catch(console.error)
+
+  if (!isSync) {
+    console.error('EBT failed to replicate! Final state:')
+    console.log(person1.id, await p(person1.getVectorClock)())
+    console.log(person2.id, await p(person2.getVectorClock)())
 
+    console.error('falling back to legacyReplicate')
+    await legacyReplicate(person1, person2)
```

Comment:
Added a fallback to legacy replication.
This function is different in that is pulls the metafeeds from person1 to person2 and vice versa (i.e. doesn't technically put the databases in the same state, as other peoples feeds will not be synced by this.)

Reply:
this didn't work out. Could make it work but it's not straight fwd